client/client-multi/client-c/listfiles/UnitTest.cmake

```diff
@@ -75,26 +81,27 @@ configure_file("${CMAKE_CURRENT_SOURCE_DIR}/sonar-project.properties.in"
         "${CMAKE_CURRENT_SOURCE_DIR}/sonar-project.properties")
 
 
-kaa_add_unit_test(NAME test_ext_log_storage_memory
+if(WITH_EXTENSION_LOGGING)
```

Comment:
I'm kind of agree but the inclusion of ext_log_storage depends on the platform. It should be moved out of core will all dependencies in the separate phase.